packages/manager/modules/resource-tagging/src/assign-tags-modal/controller.js

```diff
@@ -0,0 +1,89 @@
+import { defineCustomElement as defineCombobox } from '@ovhcloud/ods-components/dist/components/ods-combobox';
+import { TAG_REGEX } from './constants';
+
+export default class ovhManagerResourceTaggingAssignModalController {
+  /* @ngInject */
+  constructor(
+    ovhManagerResourceTaggingService,
+    $timeout,
+    $element,
+    Alerter,
+    $translate,
+  ) {
+    defineCombobox();
+    this.$timeout = $timeout;
+    this.$element = $element;
+    this.$translate = $translate;
+    this.ovhManagerResourceTaggingService = ovhManagerResourceTaggingService;
+    this.Alerter = Alerter;
+    this.loading = true;
+    this.model = {};
+  }
+
+  $onInit() {
+    this.TAG_REGEX = TAG_REGEX;
+    this.ovhManagerResourceTaggingService.getAllUserTags().then((tags) => {
+      this.tags = tags;
+      this.keys = tags.map((tag) => tag.key);
+      this.values = [];
+      this.loading = false;
+    });
+  }
+
+  updateModel(event) {
+    if (event.detail.name === 'tag-key') {
```

Comment:
```suggestion
    if (event.detail.name === TAG_KEY) {
```

Can we keep this 'tag-key' in constant ?